### ISSUE TITLE
Context of password not handled correctly for thing config parameter

### ISSUE THREAD

**Issue by mhilbush** [openhab/org.openhab.ui.habmin#247]
The following thing config parameter specification should display the information masked.
```
            <parameter name="password" type="text" required="true">
                <label>Password</label>
                <description>Password used to login to SunsetWx service</description>
                <context>password</context>
            </parameter>
```

Here's how it looks in PaperUI.
![capture](https://cloud.githubusercontent.com/assets/19286354/21022948/7c365d08-bd4c-11e6-98ce-2a4b1d3e0bc3.JPG)

**Comment by cdjackson** [comment]
I've just added code to support this (hopefully) - @mhilbush can you test please (I'll kick off a build in a few minutes).

**Comment by mhilbush** [comment]
Looks like it's still picking up the old template.

Way out of my area of knowledge...  But, is the order of the entries in eshConfigParam.tpl.html important?

Will this be selected
`<div ng-if="config.options==null || config.options.length==0">`

before this?
`<div ng-if="()config.options==null || config.options.length==0) && config.context=='password'">`

**Comment by cdjackson** [comment]
Yes - you might be right. I also see that Eclipse nicely added a closed bracket for me so the condition doesn’t work in any case. I’ll update in a few minutes.
